ballista/core/proto/datafusion.proto

```diff
@@ -1295,18 +1507,18 @@ message RepartitionExecNode {
   }
 }
 
-message JoinFilter {
+message JoinFilter{
   PhysicalExprNode expression = 1;
   repeated ColumnIndex column_indices = 2;
   Schema schema = 3;
 }
 
-message ColumnIndex {
+message ColumnIndex{
   uint32 index = 1;
   JoinSide side = 2;
 }
 
-enum JoinSide {
+enum JoinSide{
```

Comment:
Inconsistent formatting changes. There are a few more of these

Reply:
yeah, not sure we want to fix it, as this is a literal copy from upstream datafusion